e2e/cli_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e_test
+
+import (
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("CLI", func() {
+	act := func(arg ...string) (session *gexec.Session, err error) {
+		cmd := exec.Command(cliPath, arg...)
+		session, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+		return
+	}
+
+	Describe("help", func() {
+		It("should show the help for server", func() {
+			session, err := act("help")
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say(`Usage:`))
```

Comment:
How about dot-importing `gbytes` and getting rid of this qualifier just like we already do with `gomega` and its matchers? Would make the expectations read more naturally imo.